Tests/StreamChatTests/Repositories/AuthenticationRepository_Tests.swift

```diff
@@ -1103,14 +1119,18 @@ private class AuthenticationRepositoryDelegateMock: AuthenticationRepositoryDele
     var newState: EnvironmentState?
     var logoutCallCount: Int = 0
     var newStateCalls: Int = 0
+    var isCapturingStatistics = true
```

Comment:
I needed to turn off this in one of the new test, since the mock delegate is not thread-safe and otherwise would crash the test. Alternative would have been to make this thread-safe as well but I did not go for it since I do not need these statistics.